[esx_addons]/esx_lscustom/client/main.lua

```diff
@@ -1,542 +1,434 @@
-
-            if k == data.current.modType or isRimMod then
-
-                if data.current.label == _U('by_default') or string.match(data.current.label, _U('installed')) then
-                    ESX.ShowNotification(_U('already_own', data.current.label))
-										myCar = ESX.Game.GetVehicleProperties(vehicle)
-										TriggerServerEvent('esx_lscustom:refreshOwnedVehicle', myCar)
-                else
-                    local vehiclePrice = 50000
-
-                    for i = 1, #Vehicles, 1 do
-                        if GetEntityModel(vehicle) == GetHashKey(Vehicles[i].model) then
-                            vehiclePrice = Vehicles[i].price
-                            break
-                        end
-                    end
-
-                    if isRimMod then
-                        price = math.floor(vehiclePrice * data.current.price / 100)
-                        TriggerServerEvent('esx_lscustom:buyMod', price)
-                    elseif v.modType == 11 or v.modType == 12 or v.modType == 13 or v.modType == 15 or v.modType == 16 then
-                        price = math.floor(vehiclePrice * v.price[data.current.modNum + 1] / 100)
-                        TriggerServerEvent('esx_lscustom:buyMod', price)
-                    elseif v.modType == 17 then
-                        price = math.floor(vehiclePrice * v.price[1] / 100)
-                        TriggerServerEvent('esx_lscustom:buyMod', price)
-                    else
-                        price = math.floor(vehiclePrice * v.price / 100)
-                        TriggerServerEvent('esx_lscustom:buyMod', price)
-                    end
-                end
-
-                menu.close()
-                found = true
-                break
-            end
-
-        end
-
-        if not found then
-            GetAction(data.current)
-        end
-    end, function(data, menu) -- on cancel
-        menu.close()
-        TriggerEvent('esx_lscustom:cancelInstallMod')
-
-        local playerPed = PlayerPedId()
-        local vehicle = GetVehiclePedIsIn(playerPed, false)
-        SetVehicleDoorsShut(vehicle, false)
-
-        if parent == nil then
-            lsMenuIsShowed = false
-            local vehicle = GetVehiclePedIsIn(PlayerPedId(), false)
-            FreezeEntityPosition(vehicle, false)
-            myCar = {}
-        end
-    end, function(data, menu) -- on change
-        UpdateMods(data.current)
-    end)
+		end
+
+		if not found then
+			GetAction(data.current)
+		end
+	end, function(data, menu) -- on cancel
+		menu.close()
+		TriggerEvent('esx_lscustom:cancelInstallMod')
+		
+		local playerPed = PlayerPedId()
+		local vehicle = GetVehiclePedIsIn(playerPed, false)
+		SetVehicleDoorsShut(vehicle, false)
+
+		if parent == nil then
+			lsMenuIsShowed = false
+			local vehicle = GetVehiclePedIsIn(PlayerPedId(), false)
+			TriggerClientEvent('esx_lscustom:changeVehicleData', "remove", {plate = GetVehicleNumberPlateText(vehicle)})
```

Comment:
⚠ You must trigger the server, not the client!
-----------------------------------------------
You can replace this line with :

```lua
TriggerServerEvent('esx_lscustom:changeVehicleData', "remove")
```

Reply:
as
```
TriggerServerEvent('esx_lscustom:changeVehicleData', "add", myCar)
```